scripts/core/usm.yml

```diff
@@ -331,6 +331,62 @@ returns:
     - $X_RESULT_ERROR_OUT_OF_RESOURCES
 --- #--------------------------------------------------------------------------
 type: function
+desc: "USM allocate pitched memory"
+class: $xUSM
+name: PitchedAllocExp
+ordinal: "0"
+analogue:
+    - "**cuMemAllocPitch**"
+details:
+  - "This function must support memory pooling."
+  - "If pUSMDesc is not NULL and pUSMDesc->pool is not NULL the allocation will be served from a specified memory pool."
+  - "Otherwise, the behavior is implementation-defined."
+  - "Allocations served from different memory pools must be isolated and must not reside on the same page."
+  - "Any flags/hints passed through pUSMDesc only affect the single allocation."
+  - "See also $x_usm_host_desc_t."
+  - "See also $x_usm_device_desc_t."
+params:
+    - type: $x_context_handle_t
+      name: hContext
+      desc: "[in] handle of the context object"
+    - type: $x_device_handle_t
+      name: hDevice
+      desc: "[in] handle of the device object"
+    - type: const $x_usm_desc_t*
+      name: pUSMDesc
+      desc: "[in][optional] Pointer to USM memory allocation descriptor."
+    - type: $x_usm_pool_handle_t
+      name: pool
+      desc: "[in][optional] Pointer to a pool created using urUSMPoolCreate"
+    - type: "size_t"
+      name: widthInBytes
+      desc: "[in] width in bytes of the USM memory object to be allocated"
+    - type: "size_t"
+      name: height
+      desc: "[in] height of the USM memory object to be allocated"
+    - type: "size_t"
+      name: elementSizeBytes
+      desc: "[in] size in bytes of an element in the allocation"
+    - type: void**
+      name: ppMem
+      desc: "[out] pointer to USM shared memory object"
+    - type: size_t*
+      name: pResultPitch
+      desc: "[out] pitch of the allocation"
+returns:
+    - $X_RESULT_ERROR_INVALID_CONTEXT
+    - $X_RESULT_ERROR_INVALID_VALUE:
+      - "`pUSMDesc && pUSMDesc->align != 0 && ((pUSMDesc->align & (pUSMDesc->align-1)) != 0)`" # alignment must be power of two
+      - "If `align` is greater that the size of the largest data type supported by `hDevice`."
+    - $X_RESULT_ERROR_INVALID_USM_SIZE:
+      - "`widthInBytes == 0`"
+      - "`size` is greater than $X_DEVICE_INFO_MAX_MEM_ALLOC_SIZE."
+    - $X_RESULT_ERROR_INVALID_OPERATION:
+      - "If `UR_DEVICE_INFO_USM_SINGLE_SHARED_SUPPORT` and `UR_DEVICE_INFO_USM_CROSS_SHARED_SUPPORT` are both false."
+    - $X_RESULT_ERROR_OUT_OF_HOST_MEMORY
+    - $X_RESULT_ERROR_OUT_OF_RESOURCES
+--- #--------------------------------------------------------------------------
```

Comment:
This should be defined in the `source/core/exp-bindless-images.yml` file.

Reply:
Done. Removed from here.